From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Fri, 25 Sep 2026 13:27:58 -0400
Subject: [PATCH] cmake: Add missing log_add_impl.hpp to distributed headers

This is a new header for v4.11, referenced by the existing log.hpp and
log_add.hpp public headers.

Signed-off-by: Ryan Volz <ryan.volz@gmail.com>
---
 host/include/uhd/utils/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/host/include/uhd/utils/CMakeLists.txt b/host/include/uhd/utils/CMakeLists.txt
index 6c380f043..d1e3d78e7 100644
--- a/host/include/uhd/utils/CMakeLists.txt
+++ b/host/include/uhd/utils/CMakeLists.txt
@@ -24,6 +24,7 @@ UHD_INSTALL(FILES
     interpolation.hpp
     log.hpp
     log_add.hpp
+    log_add_impl.hpp
     math.hpp
     msg_task.hpp
     noncopyable.hpp
-- 
2.51.0

